README.rst

```diff
@@ -76,15 +69,13 @@ The file is YAML, it contains an object with the following members:
       contains a token string created from
       https://github.com/settings/tokens
 
-      Set it up with eg.:
+      Set it up each time with eg.:
 
       .. code:: sh
 
-         read GITHUB_TOKEN
-         export GITHUB_TOKEN
-
-      Or add it to an environment file *not under revision control*.
+         export GITHUB_TOKEN=your_github_token
```

Comment:
I was suggesting a way that doesn't leave a trace. The problem with doing that export is that it may find itself in the bash history, for a typical user.

Reply:
ah! i see. i'll clarify in the doc